tests/unit/user_interaction_test.py

```diff
@@ -0,0 +1,85 @@
+from unittest import TestCase
+import user_interaction
+from unittest.mock import patch
+import random
+import string
+
+
+class UserInteractionTest(TestCase):
+    def get_random_text(self):
+        return ''.join(random.choices(string.ascii_uppercase + string.digits, k=random.randint(1, 20)))
```

Comment:
Not sure if we should randomly generate input for print methods.
Its better to check different edge cases manually. Such as:
empty string
string with special symbols (like "{ . ," and so on)
strings with mix of symbols from different languages
numbers

Reply:
Great advice, I have done it.